python-backend/app/api/mines/mine/resources/mine.py

```diff
@@ -436,17 +451,19 @@ def _mine_status_processor(mine_status, mine):
 
     existing_status = mine.mine_status[0] if mine.mine_status else None
     if existing_status:
-        if existing_status.mine_status_xref_guid == mine_status_xref.mine_status_xref_guid:
+        if existing_status.mine_status_xref_guid == mine_status_xref.mine_status_xref_guid \
+                and status_date == existing_status.status_date:
```

Comment:
Just checking, a change to only the status_date will cause it to create a new row in the db? or should it update in place. We do want effective dating on this table, but i'm just wondering if this was intended

Reply:
Discussed offline that this was intended.